### ISSUE TITLE
module 'torch.optim.lr_scheduler' has no attribute 'LinearLR'

### ISSUE THREAD

**Issue by Rohaifa** [pytorch/vision#4521]
https://github.com/pytorch/vision/blob/972ca657416c5896ba6e0f5fe7c0d0f3e99e1087/references/detection/engine.py#L24

**Comment by NicolasHug** [comment]
Thanks for the report @Rohaifa , this was reported in https://github.com/pytorch/vision/issues/4467 already. The LinearLR scheduler is only available on the latest pytorch version which hasn't been released yet, so for now you'll need to use the pytorch-nightly release to try it out:

`conda install pytorch -c pytorch-nightly`

**Comment by Rohaifa** [comment]
Thank you so much for your help!
